### DISCUSSION TITLE
how to create vscode-like panel at the bottom of LazyVim

### DISCUSSION THREAD

**Discussion by Dmitry-N-Medvedev** [LazyVim/LazyVim#733] [Q&A] [answered]
good morning,

I would like to learn how to create a panel at the bottom of the screen. Something like the one used in VSCode.
I would be using this panel to open a couple of terminals to start long running commands.

PS: I am sure I am not the first one who was curious, but I haven't found this in the docs.

best regards,
Dmitry

**Comment by folke** [comment, marked as answer]
Look into toggle-term

**Reply by Dmitry-N-Medvedev** [reply]
thank you! I believe we are talking about this: https://github.com/akinsho/toggleterm.nvim ?

**Reply by loiccoyle** [reply]
yep
